example/BUILD.bazel

```diff
@@ -223,3 +224,18 @@ diff_test(
     file1 = "expected7",
     file2 = "actual7",
 )
+
+
+rollup_bundle(
+    name = "bundle",
+    srcs = [
+        "require_acorn.js",
+    ],
+    deps = [
+        "@npm_deps//@gregmagolan/test-b",
+        "@npm_acorn-8.4.0",
```

Comment:
is there a reason not to use `@npm_deps//acorn` here for consistency?

Reply:
not really. just wanted to make sure that it works both ways.

Reply:
also, it is a standalone `npm_import` so it is not part of the @npm_deps repo.